### DISCUSSION TITLE
Show & Tell: add cards, embed videos, customize header links, and more

### DISCUSSION THREAD

**Discussion by aikoii** [GitbookIO/community#153] [Show and Tell]
Hey everyone! 👋

We wanted to showcase the types of content made possible by GitBook, and we figured the best way to do that would be by sharing your beautiful docs 🙂. 

This week we want to highlight [Umbraco](https://docs.umbraco.com/welcome), an open-source content management system.

Here are a few things we like about their docs:

- Easy to navigate landing page built with [cards](https://docs.gitbook.com/content-creation/blocks/cards)
- Use of the landing [page layout](https://docs.gitbook.com/publishing/customization/page-layouts)
- [Embedding](https://docs.gitbook.com/content-creation/blocks/embed-a-url) YouTube videos
- Clear [page groups](https://docs.gitbook.com/content-creation/content-structure/content-in-a-space#page-groups) and loads of tutorials
- Using[ header links](https://docs.gitbook.com/publishing/customization/space-customization#navigation) to link to other relevant spaces and resources

We will be sharing some cool docs in the coming weeks!
We would love to see your own showcases if you are proud of your docs!

**Comment by Maggs-docs** [comment]
<img width="776" height="555" alt="Screenshot 2026-04-22 at 11 32 52" src="https://github.com/user-attachments/assets/abab5d52-1f17-480d-aa3f-c6f18e61022a" />
hiya, how did you achieve this view?
